test/Performance/.gitignore

```diff
@@ -0,0 +1,5 @@
+/ext/
+/*.html
+/*.csv
+/*.xml
```

Comment:
These are files that are generated during a perf test run. This particular .gitignore is intended to apply only to the subtree under test/Performance, but if *.xml is too aggressive even there then I'm happy to back it out. The only impact would be seeing untracked files after a run.